<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="Search_label">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>120</y>
      <width>181</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Choose your search methods</string>
    </property>
   </widget>
   <widget class="QLabel" name="dataset_label">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>250</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Input Dataset Size</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Normal_button">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>160</y>
      <width>100</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Normal search</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Binary_button">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>190</y>
      <width>100</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Binary search</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="Dataset_setting">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>280</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="Dataset_button">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>320</y>
      <width>141</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Generate Dataset</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="arrayState_label">
    <property name="geometry">
     <rect>
      <x>322</x>
      <y>90</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Unsorted Array</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Merge_button">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>330</y>
      <width>191</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Sort Using Merge Sort</string>
    </property>
   </widget>
   <widget class="QPushButton" name="STL_button">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>360</y>
      <width>191</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Sort Using STL sorting</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="Find_enter">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>250</y>
      <width>161</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="Find_button">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>310</y>
      <width>100</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Find it</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="array_display">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>120</y>
      <width>211</width>
      <height>201</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="status_edit">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>93</y>
      <width>201</width>
      <height>131</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>36</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
